pages/tasker/task-completed/[taskId].tsx

```diff
@@ -0,0 +1,200 @@
+
+  return (
+    <>
+      <PageHeader title="Task Completed" />
+      <Container maxWidth="md">
+        <GrayCard>
+          <Box alignSelf="center">
+            <Typography
+              variant="h4"
+              color="primary"
+              textAlign="center"
+              fontWeight={600}
+            >
+              Task Completed
+            </Typography>
+          </Box>
+        </GrayCard>
+        <GrayCard>
+          <Box alignSelf="center">
+            <Box display="flex" justifyContent="center" mt={2} mb={5}>
+              {openPosting ? (
+                <>
+                  <CircularProgress
+                    color="secondary"
+                    size={34}
+                    sx={{ mr: 1.5 }}
+                  />
+                  <Typography
+                    variant="h6"
+                    fontWeight="normal"
+                    color="primary"
+                    mr={2}
+                  >
+                    Saving rating...
+                  </Typography>
+                </>
+              ) : (
+                <>
+                  <Typography
+                    variant="h6"
+                    fontWeight="normal"
+                    color="primary"
+                    mr={2}
+                  >
+                    Rate your experience:
+                  </Typography>
+                  <StyledRating
+                    sx={{ mx: 0 }}
+                    size="large"
+                    precision={0.5}
+                    icon={<StarRoundedIcon fontSize="inherit" />}
+                    emptyIcon={<StarOutlineRoundedIcon fontSize="inherit" />}
+                    readOnly={rating > 0 && rating <= 5}
+                    value={rating}
+                    onChange={(_, value) => {
+                      if (!value) return;
+                      setRating(value);
+                    }}
+                  />
+                </>
+              )}
+            </Box>
+            <hr
+              style={{ border: `1px solid ${gigTheme.palette.secondary.main}` }}
+            />
+            <Typography
+              variant="h6"
+              fontWeight="normal"
+              color="primary"
+              textAlign="center"
+              my={5}
+            >
+              When the requester approves your submission, you will receive a
+              reward of:
+            </Typography>
+            <Box
+              borderRadius={2}
+              width="fit-content"
+              display="flex"
+              flexDirection="row"
+              alignItems="center"
+              justifyContent="center"
+              mx="auto"
+              py={1}
+              px={2}
+              sx={{
+                backgroundImage: `linear-gradient(90deg, ${gigTheme.palette.primaryCTA.primary}, ${gigTheme.palette.primaryCTA.secondary})`,
+              }}
+            >
+              <Typography color="primary" fontWeight={400} fontSize={20}>
+                {"5"} ETH
```

Comment:
Note to pull actual reward amount

Reply:
will do, i forgot about this hardcode lol